server/paths/match.ts

```diff
@@ -16,4 +20,12 @@ export default {
       search: placementRequestPath.path('beds/search'),
     },
   },
+  placementApplications: {
```

Comment:
I wonder whether this belongs in its own placementApplications file (and the same for the routes). We'll be able to review and accept placement applictions too, and I don't think match is the best place for it.

Reply:
yeah agreed